aten/src/ATen/functorch/BatchRulesHelper.h

```diff
@@ -265,13 +264,9 @@ inline void boxed_existing_bdim_all_batch_rule(
 
   // for each tensor, ensure it has a bdim and reshape it.
   for (const auto tensor_idx : c10::irange(0, tensor_inputs.size())) {
-    const auto& value = std::get<0>(tensor_inputs[tensor_idx]);
-    auto bdim = std::get<1>(tensor_inputs[tensor_idx]);
+    const auto& [value,bdim] = tensor_inputs[tensor_idx];
```

Comment:
Huh, clang format doesn't complain about the lack of space here between the two parts of the structured assignment.

Reply:
The file is excluded